src/runtime/objmodel.cpp

```diff
@@ -4285,9 +4269,28 @@ Box* rearrangeArgumentsAndCallInternal(ParamReceiveSpec paramspec, const ParamNa
     int varargs_to_positional = std::min((int)varargs_size, paramspec.num_args - positional_to_positional);
     for (int i = 0; i < varargs_to_positional; i++) {
         assert(!rewrite_args && "would need to be handled here");
-        getArg(i + positional_to_positional, oarg1, oarg2, oarg3, oargs) = incref(PySequence_Fast_GET_ITEM(varargs, i));
+        getArg(i + positional_to_positional, oarg1, oarg2, oarg3, oargs) = PySequence_Fast_GET_ITEM(varargs, i);
     }
 
+    auto clear_refs = [&]() {
```

Comment:
very minor but could you please move the `// Clear any increfs we did for when we throw an exception:` comment to here (in front of `auto clear_refs = [&]() {`)